setup.py

```diff
@@ -51,6 +51,7 @@
     install_requires=[
         "attrs >=20, <22",
         "icontract >=2.3.4, <3",
+        "pydantic",
```

Comment:
We should at least pin it to a major version (as it is done for all other requirements): `pydantic >=1, <2`.

Reply:
fixed in 3c684cf25a88b85b6cf8664ddc5096cd6e4bc2a0